getting-started.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Getting started"
+description: Get started with Coder.
+---
+
+We’re excited for you to be part of the growing community of Coder users, and we
+wanted to provide an onboarding guide to ensure you have a great experience.
+Coder’s [documentation is completely open-source](https://github.com/cdr/docs),
+so please feel free to contribute and provide feedback as you desire.
+
+## Setup and deploy Coder
+
+To get started with Coder, you’ll need to [deploy Coder](./setup/index.md) to a
+[Kubernetes cluster](./setup/kubernetes/index.md). We have documentation to
+assist you with [creating a cluster](./setup/kubernetes/index.md) if needed.
```

Comment:
I would suggest unlinking "Kubernetes cluster" since you already link to it in the next sentence. Having 3 links here may confuse readers looking where to go. However, 2 links may allow them to understand:
- if I have a cluster, I can follow the guide to deploy coder
- if I don't have a cluster, I can follow Coder's guide to create one

EDIT: I just noticed another link below to the specific helm install docs. I would recommend removing some since this guide is focused on "getting started." If you see value in having all of these different links, I think they should be labeled more clearly so someone understands
- ./setup/index.md: everything you need to know to set up coder is nested here
- ./setup/kubernetes/index.md: guides for different types of clusters
- ./setup/installation.md: after you have a cluster that meets the requirements, you can Install here.

I am concerned people will try to click these links in the order you provided them and may get confused.